aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/internal/servlet/AwsHttpApiV2ProxyHttpServletRequest.java

```diff
@@ -498,6 +492,21 @@ public AsyncContext getAsyncContext() {
         return asyncContext;
     }
 
+    @Override
+    public String getRequestId() {
+        return null;
+    }
+
+    @Override
+    public String getProtocolRequestId() {
+        return null;
```

Comment:
return empty string, implement in `AwsHttpServletRequest`

Reply:
Updated